### DISCUSSION TITLE
MC auf 6591 nicht verwendbar: Segmentation fault

### DISCUSSION THREAD

**Discussion by AkteX** [Freetz-NG/freetz-ng#689] [Allgemein]
Hallo,
bei fast jeder Aktion (z.B. Funktionstasten oder Maus) stürzt MC mit einer Segmentation fault Meldung ab. Ich habe mit einer 6591 Box und 29er Freetz-FW getestet und gehe davon aus, dass das Problem auf ähnlichen Puma Boxen auftritt.. Ist das Problem bereits bekannt?

**Das Problem scheint mit einer aktuellen Freetz-NG Version nicht mehr aufzutreten.**

**Comment by fda77** [comment]
Riecht nach pebkac :)

**Reply by AkteX** [reply]
Wenn ich die Zielkonfiguration vollständig, wie gewünscht, einstelle und das Image baue triff das Problem i.d.R. auf. Eine Änderung der Konfiguration hilft dann auch nicht mehr, irgendeine Verbesserung hervorzurufen. In diesem Fall muss ich dann make clean, make dirclean, make distclean ausführen und neu compilieren. MC scheint, in der gleichen Konfiguraiton nur ohne Aktivierung von pyLoad richtig zu funktionieren. Wenn ich Python und Pyload schrittweise aktiviere, bleibt die korrekte Funktion von MC erhalten.
Ich habe viele Versuche unternommen, irgendeinen eindeutigen Zusammenhang zu den aktivierten Paketen zu finden. Das hat leider nichts gebracht. Nur mit MC und pyLoad in der Minimalkonfiguration tritt das Problem nicht auf.
Strace hat bei der Analyse des Segmentation Fault in MC nicht geholfen.

**Comment by fda77** [comment]
Das kann schon möglich sein, vielleicht kommt ne libs dazu oder so. Man muss halt herausfinden was die Ursache ist.
